### PR TITLE
Feature/kr/search optimization

### PR DESCRIPTION
**Pull request by KanishkaRajputd**

### CODE REVIEW

File: public/assets/icons/ArrowLeft.svg:Zone.Identifier
Comment (alphainfinitus) [inline public/assets/icons/ArrowLeft.svg:Zone.Identifier:1]: do we need this file ?

File: public/assets/search/ArrowLeft.svg:Zone.Identifier
Comment (alphainfinitus) [inline public/assets/search/ArrowLeft.svg:Zone.Identifier:1]: this file can be removed

File: src/components/Search/ResultPeople.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+	return <><div className={`${className} h-[360px] ${peopleData.length > 1 && 'overflow-y-scroll'} mt-3 -mx-6`}>
+		{peopleData.map((user, index) => <a rel="noreferrer" href={`/user/${user?.username}`} target='_blank' key={index}>
+			<DelegationProfile address={user?.defaultAddress || ''} username={user?.username} isSearch = {true}
+				className={`py-8 px-9 border-[#f3f4f5] border-solid border-[1px] shadow-[0px 22px 40px -4px rgba(235, 235, 235, 0.8)] rounded-none border-b-[0px] hover:border-pink_primary hover:border-b-[1px] cursor-pointer min-h-[180px] ${index % 2 === 1 && 'bg-[#fafafb]'} ${index === peopleData.length-1 && 'border-b-[1px]'}`}/>
+		</a>)}
+	</div>
+	<div className='flex justify-center items-center py-4 px-4'>
+		<Pagination
+			defaultCurrent={1}
```
Comment (alphainfinitus) [inline src/components/Search/ResultPeople.tsx:38]: please double check if pages start from 1 and not 0

Comment (KanishkaRajputd) [reply src/components/Search/ResultPeople.tsx:38]: checked

File: public/assets/search/ArrowLeft.svg:Zone.Identifier
Comment (KanishkaRajputd) [reply public/assets/search/ArrowLeft.svg:Zone.Identifier:1]: removed

**Review by alphainfinitus — APPROVED** [review]

**Review by alphainfinitus — APPROVED** [review]

**Pull request merged**